internal/osclients/identity.go

```diff
@@ -68,6 +69,14 @@ func (c identityClient) GetProject(ctx context.Context, id string) (*projects.Pr
 	return projects.Get(ctx, c.client, id).Extract()
 }
 
+func (c identityClient) UpdateProject(ctx context.Context, id string, opts projects.UpdateOptsBuilder) (*projects.Project, error) {
+	var updatedProject projects.Project
+	if err := projects.Update(ctx, c.client, id, opts).ExtractInto(&updatedProject); err != nil {
+		return nil, err
+	}
+	return &updatedProject, nil
```

Comment:
Can't we simplify with the following?

```suggestion
	return projects.Update(ctx, c.client, id, opts).Extract()
```

Reply:
Thanks for pointing this out!